pkg/plugin/v3/api.go

```diff
@@ -173,6 +180,11 @@ func (p *createAPIPlugin) Validate() error {
 			return fmt.Errorf("multiple groups are not allowed by default, " +
 				"to enable multi-group visit kubebuilder.io/migration/multi-group.html")
 		}
+
+		if !p.config.CRDVersionCompatible(p.resource.CRDVersion) {
+			return fmt.Errorf("only one CRD version can be used for all resources, cannot add %q",
```

Comment:
It is confuse and would lead to me understand that I cannot have CRDs scaffolds with diff GKV such as 
`create api --group=example --version=v1 --kind=app` and `create api --group=example --version=v1betav1 --kind=app`

Reply:
Care to make a suggestion?

Reply:
I think that what is confuse for me is the name crd-version and webhook-version because has no indication that is related to the API. However, it is fine. I understand your motivations for that as well.